src/drive/mobile/containers/FirstUploadModal.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react'
+import Modal, { ModalContent } from 'cozy-ui/react/Modal'
+import { connect } from 'react-redux'
+import withPersistentState from '../lib/withPersistentState'
+import styles from '../styles/uploadprogression'
+import { isIos } from '../lib/device'
+
+const MINIMUM_LONG_UPLOAD_FILES_COUNT = 50
+
+class FirstUploadModal extends Component {
+  render() {
+    const { onClose } = this.props
+    const { t } = this.context
+
+    return (
+      <Modal title={t('mobile.first_sync.title')} withCross={false}>
+        <ModalContent>
+          <h4>{t('mobile.first_sync.tips')}</h4>
+          <ul>
+            <li>{t('mobile.first_sync.tip_wifi')}</li>
+            <li>{t('mobile.first_sync.tip_bed')}</li>
+            {isIos() && <li>{t('mobile.first_sync.tip_lock')}</li>}
+          </ul>
+          <p>{t('mobile.first_sync.result')}</p>
+          <button onClick={onClose} className={styles['btn--full-width']}>
+            {t('mobile.first_sync.button')}
+          </button>
+        </ModalContent>
+      </Modal>
+    )
+  }
+}
```

Comment:
Can't we use a functional component?

Reply:
I'd like to, but is there a way to get `t` from `context` with a functional component?

Reply:
Are you sure `t` is not in `props` passed as parameter?

Reply:
It's not, but actually I can inject it.